str_analysis/generate_gnomad_json.py

```diff
@@ -528,9 +569,9 @@ def add_histograms_and_compute_readviz_paths(df, gnomad_json, most_common_motif_
         num_repeats2 = int(num_repeats2)
 
         # Update histogram and scatter plot counts
-        histogram_key1 = f"{population}/{sex_karyotype}/{pcr_protocol}/{motif1}"
-        histogram_key2 = f"{population}/{sex_karyotype}/{pcr_protocol}/{motif2}"
-        scatter_plot_key = f"{population}/{sex_karyotype}/{pcr_protocol}/{motif1}/{motif2}"
+        histogram_key1 = f"{population}/{sex_karyotype}/{motif1}"
```

Comment:
do you no longer want to track pcr protocol here?

Reply:
we're no longer planning to include it in the histograms, so it's just being removed from the histogram keys